pom.xml

```diff
@@ -78,11 +78,7 @@
 			<artifactId>spring-boot-configuration-processor</artifactId>
 			<optional>true</optional>
 		</dependency>
-        <dependency>
```

Comment:
somewhere in the `pom.xml` we tell mvn what version of java to use. Please change it to java